Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -185,7 +202,7 @@ protected Task OnRowDoubleClickedCommand( DataGridRowMouseEventArgs<TItem> event
 
         protected void OnNewCommand()
         {
-            TItem newItem = NewItemCreator != null ? NewItemCreator.Invoke() : CreateNewItem();
+            var newItem = CreateNewItem();

```

Comment:
I just noticed. 
Seems like I might have messed up the merge here. I'll fix it.
(It's strange that auto-merge did not consider it.)